sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1146,7 +1146,7 @@ export interface ListSharesResponse {
     // (undocumented)
     serviceEndpoint: string;
     // (undocumented)
-    shareItems?: ShareItem[];
+    shareItems?: ShareItemInternal[];
```

Comment:
It looks like ShareItemInternal will be exposed to user?

Reply:
It's replaced by`shareItems?: ShareItem[];`.

```
export type ServiceListSharesSegmentResponse = Omit<ServiceListSharesSegmentResponseModel, "shareItems"> & {
    shareItems?: ShareItem[];
};

// @public
export type ServiceListSharesSegmentResponseModel = ListSharesResponse & ServiceListSharesSegmentHeaders & {
    _response: coreHttp.HttpResponse & {
        parsedHeaders: ServiceListSharesSegmentHeaders;
        bodyAsText: string;
        parsedBody: ListSharesResponse;
    };
};
```